setup/launch.py

```
@@ -157,12 +158,12 @@ def shutdown(self):
 
 process_manager = ProcessManager()
```

Flake8 (7.3.0)

[error] redefinition of unused 'process_manager' from line 43

(F811)